src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/music_performance/AudioVisualPublicationModal.tsx

```diff
@@ -152,6 +170,22 @@ export const AudioVisualPublicationModal = ({
                   </TextField>
                 )}
               </Field>
+              {values.mediaType.type === 'Other' ? (
```

Comment:
hm spennende, trodde man måtte bruke `MusicMediaType.Other` her for å få riktig type siden det er enum, men liker dette bedre!